addons/web/static/src/legacy/js/core/minimal_dom.js

```diff
@@ -109,16 +125,18 @@ export function makeButtonHandler(fct) {
  *         initial state
  */
 export function addButtonLoadingEffect(btnEl) {
+    // Note that pe-none is used alongside "disabled" so that this works on
+    // links not using the "btn" class.
     if (btnEl.classList) {
-        btnEl.classList.add('o_website_btn_loading', 'disabled');
+        btnEl.classList.add('o_website_btn_loading', 'disabled', 'pe-none');
```

Comment:
Same explanation as https://github.com/odoo/odoo/pull/158661#discussion_r1539746596

=> Your "before" was probably only working on custom code using the `addButtonLoadingEffect` function on link elements. Using the `makeButtonHandler` (or the `async` keyword magic in the event handlers declaration) only worked on `.btn` elements. This is a valid usecase of course to use the function by hand, but on `.btn` elements it did not work because of the `disabled` class we are adding: I'd rather have the same behavior for both and the easiest would be to remove the cursor rule and not mess with the bootstrap `pointer-events` rule.

Reply:
> Your "before" was probably only working on custom code using the addButtonLoadingEffect function on link elements.

Yes, for the screenshot I made it was done for instance by the `s_website_form/000.js`:
```js
const $button = this.$target.find('.s_website_form_send, .o_website_form_send');
$button.addClass('disabled') // !compatibility
       .attr('disabled', 'disabled');
this.restoreBtnLoading = dom.addButtonLoadingEffect($button[0]);
```
Just need to turn the button into a link and then it shows as in the screenshot, but no big deal, no worth trying to fight bootstrap about it for a behavior which was broken on button anyway